ui/src/app/cohort-search/overview/overview.component.html

```diff
@@ -9,11 +9,12 @@ <h2 id="total-count">
         {{ total$ | async | number }}
       </span>
     </h2>
-    <div>
+    <div *ngIf="chartData$ | async as chartData">
       <button
         type="button"
         class="btn btn-primary"
         [style.margin-left]="'0'"
+        [disabled]="!chartData.size || (total$ | async | number) === 0"
```

Comment:
Do we need the number pipe here?

Reply:
Its working without number pipe. so I will remove number pipe.